frontend/constants/model-permissions.ts

```diff
@@ -17,14 +17,9 @@ export const MODEL_ROLES_DESC = {
 
 export const MODEL_ROLES_LIST = [
 	{
-		key: MODEL_ROLES_TYPES.UNASSIGNED,
-		label: 'Unassigned',
-		tooltip: MODEL_ROLES_DESC.UNASSIGNED
-	},
-	{
-		key: MODEL_ROLES_TYPES.VIEWER,
-		label: 'Viewer',
-		tooltip: MODEL_ROLES_DESC.VIEWER
+		key: MODEL_ROLES_TYPES.ADMINISTRATOR,
```

Comment:
I don't think you want to shift this order.

The order of this is intention by level of permissions

Reply:
Hi @carmenfan , i have put these back into their original order 👍

Reply:
> ![image](https://user-images.githubusercontent.com/11945337/55318535-e9f62500-546a-11e9-9f99-8bb53f5685bd.png)
> 
> Models in new federation tab, project in new model tab and project in user permissions tab should be sorted

I have resolved these lists carmen, it required additional sorting on the teamspaces.redux as well as the userManagement. 

Unlike the jobs.redux which only had to be done in one place. :)